x-pack/plugin/esql/qa/testFixtures/src/main/resources/exponential_histogram.csv-spec

```diff
@@ -13,3 +13,32 @@ dummy-positive_only       | "{""scale"":2,""sum"":1275.0,""min"":1.0,""max"":50.
 dummy-zero_count_only     | "{""scale"":2,""sum"":0.0,""min"":0.0,""max"":0.0,""zero"":{""count"":101}}"
 dummy-zero_threshold_only | "{""scale"":0,""sum"":0.0,""zero"":{""threshold"":2.0E-5}}"
 ;
+
+ungroupedPercentiles
+required_capability: exponential_histogram
+
+FROM exp_histo_sample | WHERE NOT STARTS_WITH(instance, "dummy") 
+ | STATS p0 = PERCENTILE(responseTime,0),  p50 = PERCENTILE(responseTime,50),  p99 = PERCENTILE(responseTime, 99),  p100 = PERCENTILE(responseTime,100)
+ | EVAL p50 = ROUND(p50, 7), p99 = ROUND(p99, 7) // rounding to avoid floating point precision issues, min and max are exact so no rounding needed
```

Comment:
I had an interesting test failure where the percentiles where a tiny, tiny bit off the exact values on CI.
This also wouldn't reproduce locally with the same seed.
My guess here is that this is due to the different implementation of Math.log / Math.exp on my machine (Mac M1) vs the CI runners. See also [this SO thread](https://stackoverflow.com/questions/74074312/standard-math-functions-reproducibility-on-different-cpus)

For that reason I've added the rounding to make the tests more robust.